### PR TITLE
add networkobservability scrape config

### PR DESCRIPTION
**Pull request by matmerr**
Kappie has been renamed to Retina, this provides a secondary config for migration

### CODE REVIEW

File: otelcollector/configmapparser/tomlparser-default-targets-metrics-keep-list.rb [hunk 1]
```diff
@@ -44,6 +45,7 @@
...
 @nodeexporterRegex_minimal = "node_filesystem_readonly|node_cpu_seconds_total|node_memory_MemAvailable_bytes|node_memory_Buffers_bytes|node_memory_Cached_bytes|node_memory_MemFree_bytes|node_memory_Slab_bytes|node_memory_MemTotal_bytes|node_netstat_Tcp_RetransSegs|node_netstat_Tcp_OutSegs|node_netstat_TcpExt_TCPSynRetrans|node_load1|node_load5|node_load15|node_disk_read_bytes_total|node_disk_written_bytes_total|node_disk_io_time_seconds_total|node_filesystem_size_bytes|node_filesystem_avail_bytes|node_network_receive_bytes_total|node_network_transmit_bytes_total|node_vmstat_pgmajfault|node_network_receive_drop_total|node_network_transmit_drop_total|node_disk_io_time_weighted_seconds_total|node_exporter_build_info|node_time_seconds|node_uname_info|kubernetes_build_info"
 @kappiebasicRegex_minimal = "kappie.*"
+@retinaRegex_minimal = "retina.*"
```
Comment (vakalapa) [inline otelcollector/configmapparser/tomlparser-default-targets-metrics-keep-list.rb]: need this to be networkobservability_

File: otelcollector/configmapparser/tomlparser-default-scrape-settings.rb [hunk 2]
```diff
@@ -22,6 +22,7 @@
 @windowsexporterEnabled = false
 @windowskubeproxyEnabled = false
 @kappiebasicEnabled = true
+@networkobservabilityEnabled = false
```
Comment (matmerr) [inline otelcollector/configmapparser/tomlparser-default-scrape-settings.rb:25]: true?

File: otelcollector/configmapparser/default-prom-configs/networkobservabilityDefaultDs.yml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        action: replace
+
+      - source_labels: [__address__, __meta_kubernetes_service_port_number]
+        action: replace
+        regex: ([^:]+)(?::\d+)?;(\d+)
+        replacement: $1:$2
+        target_label: __address__
+
+      - source_labels: [__address__]
+        replacement: "$NODE_NAME"
```
Comment (vishiy) [inline otelcollector/configmapparser/default-prom-configs/networkobservabilityDefaultDs.yml:28]: please make $NODE_IP --> $$NODE_IP$$, and also the same for NODE_NAME (for all 3 jobs)

File: otelcollector/telegraf/telegraf-prometheus-collector.conf [hunk 4]
```diff
@@ -170,6 +170,7 @@
     defaultscrapepodannotations = "$AZMON_PROMETHEUS_POD_ANNOTATION_SCRAPING_ENABLED"
     podannotationns = "$AZMON_PROMETHEUS_POD_ANNOTATION_NAMESPACES_REGEX"
     defaultscrapekappiebasic = "$AZMON_PROMETHEUS_KAPPIEBASIC_SCRAPING_ENABLED"
+    defaultscrapenetworkobservabilitybasic = "$AZMON_PROMETHEUS_NETWORKOBSERVABILITY_SCRAPING_ENABLED"
```
Comment (vishiy) [inline otelcollector/telegraf/telegraf-prometheus-collector.conf:173]: no basic ?

File: otelcollector/configmapparser/tomlparser-default-targets-metrics-keep-list.rb [hunk 5]
```diff
@@ -275,7 +293,7 @@ def populateRegexValuesWithMinimalIngestionProfile
         @apiserverRegex = @apiserverRegex + "|" + @apiserverRegex_minimal
         @kubestateRegex = @kubestateRegex + "|" + @kubestateRegex_minimal
         @nodeexporterRegex = @nodeexporterRegex + "|" + @nodeexporterRegex_minimal
-        @kappiebasicRegex = @kappiebasicRegex + "|" + @kappiebasicRegex_minimal
+        @networkobservabilityRegex = @networkobservabilityRegex + "|" + @networkobservabilityRegex_minimal
```
Comment (matmerr) [inline otelcollector/configmapparser/tomlparser-default-targets-metrics-keep-list.rb:296]: Keep Kappie basic

Comment (vishiy) [inline otelcollector/configmapparser/tomlparser-default-targets-metrics-keep-list.rb:296]: put back kappiebasic

**Pull request closed without merging**